public/app/features/plugins/extensions/registry.ts

```diff
@@ -0,0 +1,36 @@
+import type { PluginsExtensionConfig, PluginsExtensionLinkConfig } from '@grafana/data';
+import type { PluginsExtensionRegistry, PluginsExtensionRegistryLink } from '@grafana/runtime';
+
+export function configurePluginExtensions(
```

Comment:
Nit: maybe calling it something like `createPluginExtensionsRegistry()` would communicate its intentions better?